### ISSUE TITLE
Rechte/Lizenzen der E-Texte

### ISSUE THREAD

**Issue by wachauer** [Middle-High-German-Conceptual-Database/MHDBDB#65]
Siehe Excel-Sheet unter
\MHDBDB\Relaunch\Rechtliches

(work in progress durch @thilielx )

Nach Fertigstellung durch Alan musst du, @KardungLa, die Spalte "Rechtslage manuell" auslesen und bitte wie folgt anwenden:

"open" = Volltext lesen, Volltext downloaden
"limited" = Volltext lesen, nicht downloaden
"closed" = weder lesen noch downloaden, nur in Auszügen (via Belegsuche etc.), gilt als Zitat

**Comment by wachauer** [comment]
@clemdoc schau mal, ob du oberen Sharepoint-Link anklicken kannst, da ist alles drin, was du für die rechtlichen Download-Fragen brauchst.

**Comment by clemdoc** [comment]
@wachauer Nein, klappt leider nicht. Wird dann sicher mit dem externen Zugriff funktionieren.
